keps/0004-add-testing-infrastructure.md

```diff
@@ -89,7 +89,7 @@ Go coverage reports will be used to ensure that changes do not reduce test cover
 
 #### Integration tests
 
-Integration tests test the KUDO controller in its entirety. These tests can be run against either a real Kubernetes cluster or a local control plane. Integration tests will be manually run after review, but prior to merging using the [Circle CI manual approval feature](https://circleci.com/docs/2.0/triggers/#manual-approval). Integration tests can be written using the test harness designed in KEP-0008.
+Integration tests test the KUDO controller in its entirety. These tests can be run against either a real Kubernetes cluster or a local control plane. Integration tests will be manually run after review, but prior to merging using `/test` command supported by Prow. Integration tests can be written using the test harness designed in KEP-0008.
```

Comment:
Isn't this automatic?  The statement

> Integration tests will be manually run after review, but prior to merging using `/test` command supported by Prow.

Does not give this impression.

Reply:
For the most part yes, but the integration test jobs we *want* to manually trigger on the PR as needed (and run on master). We can configure Prow this which, which is what I'm talking about in this paragraph.

Reply:
Oops, actually, it's the E2E tests that should be explicitly run.. will update.